Description: Allow direct LIB_INST_DIR override.
Author: Mihai Moldovan <ionic@ionic.de>

--- a/libcomps/src/CMakeLists.txt
+++ b/libcomps/src/CMakeLists.txt
@@ -51,10 +51,12 @@ set_target_properties(libcomps PROPERTIE
 
 add_dependencies(libcomps src-copy)
 
+IF (NOT LIB_INST_DIR)
 IF (CMAKE_SIZEOF_VOID_P MATCHES "8")
     SET (LIB_SUFFIX "64")
 ENDIF (CMAKE_SIZEOF_VOID_P MATCHES "8")
 set (LIB_INST_DIR ${CMAKE_INSTALL_LIBDIR})
+ENDIF(NOT LIB_INST_DIR)
 
 
 install (FILES ${libcomps_HEADERS} DESTINATION include/libcomps)
